vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/search/CheckboxNameSearch.java

```diff
@@ -77,7 +80,10 @@ private List<WebElement> searchCheckboxByLabels(SearchContext searchContext, Sea
             }
             else
             {
-                checkboxes = label.findElements(getXPathLocator(PRECEDING_SIBLING_CHECKBOX_LOCATOR));
+                checkboxes = Stream
```

Comment:
What about integration tests update? :)

Reply:
integration test should work fine with this change ;)

Reply:
that's true, but we should add integration tests to cover new logic